bsdlib/include/nrf_socket.h

```diff
@@ -184,15 +185,36 @@ typedef int32_t ssize_t;
 #define NRF_GNSS_SV_FLAG_UNHEALTHY    8 /**< Indicate that the satellite is unhealthy. */
 /**@} */
 
+/**@defgroup nrf_gnss_agps_data_type Flags indicating which AGPS assistance data set is written to the GPS module
```

Comment:
These values are are not flags, but enumerator values.

Reply:
```suggestion
/**@defgroup nrf_gnss_agps_data_type Values setting which AGPS assistance data set is written to the GPS module
```

Reply:
Seems like I just commented on a merged PR :)